docs/how-to/topics/file-management.rst

```diff
@@ -0,0 +1,206 @@
+
+A second possible option might be to put the file in ``configfilecreator/src/configfilecreator`` because that's where the ``app.py`` file is. After all, Python bases file access on the directory from which the program is being run. While the second option does ensure Briefcase packages the file with the app, it still doesn't guarantee a consistent path.
+
+To avoid the possibility of either of the above happening accidentally, Briefcase sets the current working directory to elsewhere so we aren't caught by this issue.
+
+Let's take a look at an example of packaging the file content with the app, and using ``__file__`` to locate it.
+
+Create an ``initial_config.toml`` file containing the following content, and place it in the ``resources/`` directory within the Briefcase project:
+
+.. code-block:: toml
+
+    # Update the following to match your configuration
+    PROJECT_NAME = "Name"
+    PROJECT_VERSION = "v0.0.0"
+    AUTHOR_NAME = "Your Name"
+
+Update the ``load_button_pressed`` handler to the following:
+
+.. code-block:: python
+
+    def load_button_pressed(self, button, **kwargs):
+        path = Path(__file__).parent / "resources" / "initial_config.toml"
+        self.text_input.value = path.read_text(encoding="utf-8")
+
+Now when you run the app, it will start, and the button will successfully load the contents of the file into the multi-line text input. This works in this situation, because we have only one app file, and the standard directory structure. However, in a more complex application with multiple levels of modules, or when calling a library that is independent of the app, to use ``Path(__file__)``, we would need to know where the file that is reading the code is in the code checkout relative to the ``resources`` directory.
+
+So, how do we get the benefits of absolute paths, but ensure that the file can be found regardless of where the app is being run? This is where Toga can help.
+
+App paths
+=========
+
+Toga includes an :doc:`app paths <../../reference/api/resources/app_paths>` feature that provides a selection of known locations on the user's computer. Provided as ``pathlib.Path`` objects, they are known-safe locations for reading and writing files, that are specific to each operating system. Each user running an application will have their own unique app paths.
+
+The read-only path location, ``paths.app``, provides an anchor from the location of the app file. [#f1]_ It can therefore be used to construct absolute paths based on the app file location within the package. For this to work, we need to package the file with our app. Briefcase guarantees that any file in the project directory (``configfilecreator/src/configfilecreator`` in the example project structure shown above), will be included with the packaged app, including the contents of any subdirectories. There are other ways to ensure a file is included - see the :doc:`Source <../../reference/api/resources/sources/source>` documentation for details.
```

Comment:
The reference to Briefcase's packaging behavior has been referred to indirectly above; might want to restructure that section further up the page to the point where we're creating the initial_config.toml content.

Reply:
The `sources` reference her his a different thing - I was referring to the Briefcase setting. Toga's `sources` are dynamic sources of data for lists, tables, trees etc.